docs/index.md

```diff
@@ -96,4 +91,4 @@ testing DSL.
   Inferno's Ruby API.
 
 ## Inferno Test Kits
-Visit the [Community Page](/inferno-core/available-test-kits) to view all available Test Kits.
+Visit the [Test Kits](https://inferno.healthit.gov/test-kits/) page to view all available Test Kits.
```

Comment:
```suggestion
Visit the [Community Test Kits](/community/test-kits.html) page to view all available Test Kits.
```

Something that is a little confusing is that inferno.healthit.gov does not host all test kits.  Its just one 'Test Kit Platform' that ONC is paying hosting fees for.  It cannot be the only place you can run test kits though, as we aren't really in the SaaS/PaaS business.  So we maintain a separate list of test kits on this site.  Which, yeah, is redundant for the most part, but not quite completely, especially if more people use inferno to create testing solutions.